src/etl/bgi_etl.py

```diff
@@ -177,333 +183,443 @@ def _load_and_process_data(self):
-
-                                if page == 'gene':
-                                    modCrossReferenceCompleteUrl = ETLHelper.get_page_complete_url(localCrossRefId, ETL.xrefUrlMap, prefix, prefix + page)
-
-                                geneticEntityExternalUrl = ETLHelper.get_page_complete_url(localCrossRefId, ETL.xrefUrlMap, prefix, prefix + page)
+                                    cross_ref_complete_url = ETLHelper.get_expression_images_url(\
+                                            local_cross_ref_id,
+                                            cross_ref_id)
+                                elif page == 'gene':
+                                    mod_cross_reference_complete_url \
+                                        = ETLHelper.get_page_complete_url(local_cross_ref_id,
+                                                                          ETL.xref_url_map,
+                                                                          prefix,
+                                                                          prefix + page)
+
+                                genetic_entity_external_url = ETLHelper.get_page_complete_url(\
+                                        local_cross_ref_id,
+                                        ETL.xref_url_map,
+                                        prefix,
+                                        prefix + page)
 
                                 if page == 'gene/references':
-                                   # if crossRefId.startswith('ZFIN'):
-                                       # logger.info(crossRefId + " " + page)
-                                    geneLiteratureUrl = ETLHelper.get_page_complete_url(localCrossRefId, ETL.xrefUrlMap, prefix, prefix + page)
+                                    gene_literature_url = ETLHelper.get_page_complete_url(\
+                                            local_cross_ref_id,
+                                            ETL.xref_url_map,
+                                            prefix,
+                                            prefix + page)
 
                                 if page == 'gene/spell':
-                                    displayName='Serial Patterns of Expression Levels Locator (SPELL)'
+                                    display_name = \
+                                            'Serial Patterns of Expression Levels Locator (SPELL)'
 
                                 # TODO: fix generic_cross_reference in SGD, RGD
 
                                 if page == 'generic_cross_reference':
-                                    crossRefCompleteUrl = ETLHelper.get_no_page_complete_url(localCrossRefId, ETL.xrefUrlMap, prefix, primary_id)
-
-                                # TODO: fix gene/disease xrefs for SGD once resourceDescriptor change in develop
+                                    cross_ref_complete_url = ETLHelper.get_no_page_complete_url(\
```

Comment:
can the first item in the list be in line with the function call, I find this more readable than the \.